includes/options/class-amp-options-menu.php

```diff
@@ -318,9 +319,13 @@ public function render_theme_support() {
 						<p><?php esc_html_e( 'Your active theme is known to work well in standard or transitional mode.', 'amp' ); ?></p>
 					</div>
 				<?php endif; ?>
+			<?php elseif ( AMP_Theme_Support::supports_reader_mode() ) : ?>
+				<div class="notice notice-success notice-alt inline">
+					<p><?php echo wp_kses( $reader_mode_support, [ 'strong' => [] ] ); ?></p>
+				</div>
 			<?php endif; ?>
 
-			<?php if ( ! AMP_Theme_Support::get_support_mode_added_via_theme() ) : ?>
```

Comment:
For consistency, I'm also including `! $builtin_support` in the conditions here. See ffee40fd. In reality `AMP_Theme_Support::get_support_mode_added_via_theme()` should actually return `true` when `in_array( get_template(), AMP_Core_Theme_Sanitizer::get_supported_themes(), true )`.

We'll want to revisit how we help users discover AMP-compatible themes in https://github.com/ampproject/amp-wp/issues/2313.